src/Lookup/InMemoryEntityLookup.php

```diff
@@ -33,6 +37,17 @@ public function addEntity( EntityDocument $entity ) {
 	}
 
 	/**
+	 * If an exception with the same EntityId was already present it will be replaced by the new one.
```

Comment:
A basic explanation of what the function does. Currently this only describes a side effect. E.g. "Collects exceptions that should be thrown when a entity with the id in the exception is requested."